class UserNotFoundError(Exception):
    pass


class VerifyPasswordError(Exception):
    pass
